src/graph/executor/Executor.cpp

```diff
@@ -1,7 +1,6 @@
-/* Copyright (c) 2020 vesoft inc. All rights reserved.
- *
- * This source code is licensed under Apache 2.0 License.
- */
+// Copyright (c) 2022 vesoft inc. All rights reserved.
```

Comment:
maybe should not update the year info. It represent the file create time and maybe make reader confuse.

Reply:
Agree.

Reply:
done